packages/cli/src/commands/deploy/latest.ts

```diff
@@ -176,7 +189,7 @@ const printDeploymentStatus = async (
 };
 
 // Converts `env` Arrays, Strings and Objects into env Objects.
-const parseEnv = env => {
+const parseEnv = (env: any | object) => {
```

Comment:
I think the type here should be `env: Env | undefined` which you can find from `@vercel/build-utils`

Reply:
@styfle I get the following error. I think that is expect an array 🤔 
<img width="525" alt="Screen Shot 2021-06-23 at 12 13 06" src="https://user-images.githubusercontent.com/16585386/123139952-60044f00-d41c-11eb-8668-eeb57c3701fb.png">

Reply:
Perhaps its `Env | string | undefined` then?

Reply:
@styfle It seems not working. Too, I tried with this variation `Env | string | string[] | undefined`

Reply:
what can we do in this case ? 🤔